### DISCUSSION TITLE
/usr/local/bin/docker-compose: exec: line 91: docker: not found

### DISCUSSION THREAD

**Discussion by ermayanka** [mrlt8/docker-wyze-bridge#104] [Q&A]
Guys - I am getting the below. Any help will be appreciated. I am using HASS.IO image on Pi. Configured docker add-on. All steps were good until the one below. Any help will be appreciated.

My technical level is developing, consider me more of a beginner. Sorry to bother but please provide detailed instructions or references when you reply. This way I will be able to follow better. Thanks a bunch!!

docker-wyze-bridge $ docker-compose up --build
/usr/local/bin/docker-compose: exec: line 91: docker: not found

**Comment by mrlt8** [comment]
If you're using the bridge as a home assistant add-on, then you should be able to start/stop from inside home assistant.

https://github.com/mrlt8/docker-wyze-bridge/wiki/Home-Assistant

**Comment by ermayanka** [comment]
Thank you. It worked.
